contrib/demo/clusters/kind/us-east1.config

```diff
@@ -12,10 +12,3 @@ nodes:
     nodeRegistration:
       kubeletExtraArgs:
         node-labels: "ingress-ready=true"
-  extraPortMappings:
```

Comment:
no need to remove those, as it could be useful to go directly against the cluster ingress in some cases.